### DISCUSSION TITLE
Transcription speed way down

### DISCUSSION THREAD

**Discussion by deantaylormax** [deepgram/community#678] [General help]
Just changed from using lambda function on AWS to transcribe files from S3 to uploading files from laptop to Deepgram API and the transcription speed is about 90% slower.  Why?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by deantaylormax** [reply]
Using this api with parameters. 

end_pt = "https://api.deepgram.com/v1/listen?model=general&tier=nova&diarize=true&paragraphs=true"
        
        headers = {
            "accept": "application/json",
            "Authorization": f'Token {DEEPGRAM_API_KEY}'
        }

**Comment by jkroll-deepgram** [comment]
Hi @deantaylormax, I assume you used the same URL parameters for both calls.

If you are working with large files, uploading files from a local laptop may be significantly slower than on AWS if you have slower internet / lower upload bandwidth?
